extensions/Rules/template/en/default/admin/params/rules.html.tmpl

```diff
@@ -0,0 +1,20 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%
+   title = "Rules"
+   desc = "Configure Change Field Rules"
+%]
+
+[%
+   param_descs = {
+     change_field_rules_enabled => 'Enable Change Field Rules',
+     change_field_rules         => 'Change Field Rules (TOML). See <a href="https://github.com/toml-lang/toml">' _
+                                   'this repo</a> for more information about TOML.'
```

Comment:
Instead of linking to the TOML language specs, we should link to the specs for the rules ext: https://docs.google.com/document/d/1KFZ7t15kEU-6DQsuSRoZrlUBeV6M2TqnedK2FNTXrhA

Reply:
> 
> 
> Instead of linking to the TOML language specs, we should link to the specs for the rules ext: https://docs.google.com/document/d/1KFZ7t15kEU-6DQsuSRoZrlUBeV6M2TqnedK2FNTXrhA

I added the docs link that will be added once pr 1591 is merged instead of the google doc.